field-library/README.md

```diff
@@ -0,0 +1,15 @@
+<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+
+# Miranj Craft Boilerplate
+
+A bunch of ready to use fields provided by Miranj to speed-up your project development process. It uses the [Field Manager](https://github.com/verbb/field-manager) plugin to import the fields.
+
+## Usage
+
+1.  Open the Control Panel and go to _Field Manager_. ![Image for step 1](images/step_1.png 'Step 1')
+2.  Go to _import_ tab in _Field Manager_.
+3.  Paste the content of _export.json_ file from the folder that you want to import.
+4.  Click on the _Configure Import_ button at the bottom of panel. ![Image for step 2](images/step_2.png 'Step 2')
+5.  Choose the _Field Group_ where you want to import the fields or add a _New Field Group_ by clicking the button at the bottom right of the panel. ![Image for step 3](images/step_3.png 'Step 3')
+6.  Click on the _import_ button at the top right of the panel to import the fields. ![Image for step 4](images/step_4.png 'Step 4')
```

Comment:
No screenshot needed.

Reply:
Removed all screenshots.

Reply:
I actually meant no screenshot needed just for _Step 4_. 🙊

Reply:
Oh okay :sweat_smile:, added the _step 2_ and _step 3_ images back and linked the _import_ in the description text to plugin documentation for import.